pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -355,3 +365,233 @@ func (c *Cluster) selectorLabels(activeDaemon string) map[string]string {
 	}
 	return labels
 }
+
+//PrometheusRuleCustomized keep customized alerts
+type PrometheusRuleCustomized struct {
+	Alerts Alerts `yaml:"alerts"`
+}
+
+//CephMgrIsAbsent alert
+type CephMgrIsAbsent struct {
```

Comment:
These types are almost all the same. Can we get rid of the duplication by reducing to a handful of types? Seems like they would be called CephAlert, CephNamespacedAlert, CephLimitAlert, etc.

Reply:
done